### PR TITLE
chore(release): update version to 1.7.0

### PR DESCRIPTION
**Pull request by kmova**
- Update to 1.7.0 version
- Enable MountChangeDetection by default

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: deploy/ndm-operator.yaml [hunk 1]
```diff
@@ -520,8 +520,7 @@ spec:
         - --feature-gates="GPTBasedUUID"
         - --feature-gates="APIService"
         # Detect changes to device size, filesystem and mount-points without restart.
-        # Uncomment the line below to enable the feature.
-        # - --feature-gates="ChangeDetection"
+        - --feature-gates="MountChangeDetection"
```
Comment (akhilerm) [inline deploy/ndm-operator.yaml]: This need not be changed, `ChangeDetection` is the correct flag name.

File: deploy/yamls/node-disk-manager.yaml [hunk 2]
```diff
@@ -35,8 +35,7 @@ spec:
         - --feature-gates="GPTBasedUUID"
         - --feature-gates="APIService"
         # Detect changes to device size, filesystem and mount-points without restart.
-        # Uncomment the line below to enable the feature.
-        # - --feature-gates="ChangeDetection"
+        - --feature-gates="MountChangeDetection"
```
Comment (akhilerm) [inline deploy/yamls/node-disk-manager.yaml]: Same comment as above

File: deploy/helm/charts/values.yaml [hunk 3]
```diff
@@ -130,7 +130,7 @@ featureGates:
     enabled: false
     featureGateFlag: "UseOSDisk"
   MountChangeDetection:
-    enabled: false
+    enabled: true
     featureGateFlag: "MountChangeDetection"
```
Comment (akhilerm) [inline deploy/helm/charts/values.yaml]: rename the flag to ChangeDetection.

File: deploy/helm/charts/values.yaml [hunk 4]
```diff
@@ -130,8 +130,8 @@ featureGates:
     enabled: false
     featureGateFlag: "UseOSDisk"
   MountChangeDetection:
```
Comment (akhilerm) [inline deploy/helm/charts/values.yaml]: @kmova can we change this also to ChangeDetection. The same change need to be done in the daemonset template also.

File: deploy/helm/charts/README.md [hunk 5]
```diff
@@ -65,7 +65,7 @@ The following table lists the configurable parameters of the OpenEBS NDM chart a
 | `ndmExporter.clusterExporter.metricsPort`   | The TCP port number used for exposing NDM cluster exporter metrics  | `9100`          |
 | `featureGates.APIService.enabled`       | Enable the gRPC API service of NDM            | `false`                                   |
 | `featureGates.UseOSDisk.enabled`        | Enable feature-gate to use free space on OS disk | `false`                                   |
-| `featureGates.MountChangeDetection.enabled` | Enable feature-gate to detect mountpoint/filesystem changes | `false`                                   |
+| `featureGates.ChangeDetection.enabled` | Enable feature-gate to detect mountpoint/filesystem changes | `true`                                   |
```
Comment (z0marlin) [inline deploy/helm/charts/README.md]: nit: this flag also enables size change detection
```suggestion
| `featureGates.ChangeDetection.enabled` | Enable feature-gate to detect mountpoint/filesystem/size changes | `true`                                   |
```

**Review by z0marlin — DISMISSED** [review]

**Comment by z0marlin** [comment]
@kmova one nit-pick comment. The rest LGTM.

**Review by akhilerm — APPROVED** [review]
LGTM

File: deploy/helm/charts/README.md (hunk 5 above)
Comment (akhilerm) [reply deploy/helm/charts/README.md]: I have accepted this suggestion into the PR.

**Pull request merged**